engine/src/main/java/org/hibernate/validator/constraintvalidation/HibernateConstraintValidatorContext.java

```diff
@@ -106,4 +110,19 @@
 	 * @since 5.3
 	 */
 	HibernateConstraintValidatorContext withDynamicPayload(Object payload);
+
+
+	/**
+	 * Returns a {@link ScriptEvaluator} created based on {@link ScriptEvaluatorFactory}
+	 * passed at bootstrap.
+	 *
+	 * @param languageName The name of a scripting language
+	 * @return A script executor for the given language. Never null.
+	 *
+	 * @throws ScriptException in case when no {@link ScriptEvaluator} was
+	 * found for a given {@code languageName}.
+	 *
+	 * @since 6.1
+	 */
+	ScriptEvaluator getScriptEvaluatorForLanguage(String languageName) throws ScriptException;
```

Comment:
Mmmmh, should it be a `ScriptException`? I thought we were trying to get away from `javax.script` specific exceptions?